core/commonMain/src/kotlinx/serialization/descriptors/SerialDescriptor.kt

```diff
@@ -162,9 +162,8 @@ public interface SerialDescriptor {
     public val isNullable: Boolean get() = false
 
     /**
-     * Returns `true` if this descriptor describes a serializable inline class.
+     * Returns `true` if this descriptor describes a serializable value class.
      */
-    @ExperimentalSerializationApi
     public val isInline: Boolean get() = false
```

Comment:
This is the only part that bothers me -- `isValue` is definitely a wrong name here as well.

WDYT about `isValueClassDescriptor`?